### DISCUSSION TITLE
我使用v5版的swiper，提示The children of `Swiper` must be `Swiper.Item` components，源码类型判断出错，求解决

### DISCUSSION THREAD

**Discussion by chenmeiling1997** [ant-design/ant-design-mobile#5288] [Q&A]
项目代码如下：
<img width="626" alt="image" src="https://user-images.githubusercontent.com/53749773/172528230-95e25088-35bf-436c-90c3-2aba88a889f5.png">
<img width="530" alt="image" src="https://user-images.githubusercontent.com/53749773/172528290-4e155afe-6ed6-4e4d-8628-d7e2dcfc88b0.png">
控制台提示如下：
![image](https://user-images.githubusercontent.com/53749773/172528347-0566e710-d8de-4200-a017-62565d398164.png)

**我查看源码得知在swiper.js会判断，child类型，于是添加日志，发现问题如下：**
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/53749773/172529016-9d522e17-8cb6-4219-ad59-2ebe0289cda5.png">

**我发现swiper里面获取到的`Swiper.Item`标签类型，与定义的类型匹配不上**
这里获取到的type是ProxyFacade(props, context)，所以报错

![image](https://user-images.githubusercontent.com/53749773/172529145-ed9535a7-500a-41a0-8788-c531f8a26707.png)

请问是我的用法写错了吗，还是什么原因呢

**Comment by miracles1919** [comment]
但看截图是没问题的，要么在 [codesandbox](https://codesandbox.io/s/antd-mobile-snrxr?file=/package.json) 或 [stackblitz](https://stackblitz.com/edit/antd-mobile?file=index.tsx) 写个复现 demo？

**Comment by MrWang2016** [comment]
遇到同一个问题，有解决吗？  @chenmeiling1997

**Reply by crazyleaves** [reply]
如果本地开启了 webpack 的热更新，可以关闭后再试一下。
